CONTRIBUTING.md

```diff
@@ -43,6 +43,40 @@ Documentation found on the [Microcosm site](http://code.viget.com/microcosm) is
 
 You may want to run what you see on [code.viget.com/microcosm](http://code.viget.com/microcosm) locally. To do so head over the [site section of this repo](site).
 
+## Prettier
+
+We are using [prettier](https://github.com/prettier/prettier) combined with [eslint](http://eslint.org/) to keep formatting and format linting easy. We do that by running:
+
+```bash
+yarn run pretty
+```
+
+or
+
+```bash
+make pretty
+```
+
+These will first run `prettier` to format our code and then run `eslint --fix` to make additional style changes and fixes.
+
+### Things to note about the prettier/eslint combo
+
```

Comment:
@nhunzaker this is new. While changing the npm script for `lint` I ran into some problems which led me to change some things. I summarized in this bit of the docs.